ci/jest-reporter.js

```diff
@@ -1,47 +1,98 @@
+const { generateDiscordCloudwatchLogFile, listECSTasks, sendDiscordNotification } = require('./helpers')
+
 const { BaseReporter } = require('@jest/reporters')
 const child_process = require('child_process')
 
+async function main_task() {
+
+  taskArns = await listECSTasks()  // like taskArns = ['arn:aws:ecs:us-east-2:967314784947:task/ceramic-dev-tests/2466935a544f47ec9a1c3d8add235c84']
+
+  if (taskArns.length > 1) {
+    console.warn('WARN: NEEDS INVESTIGATION, more than one task running !!!')
```

Comment:
This logic was borrowed from the anchor service and was not sure whether it would be needed here, but after the test runs it's clear it's not needed, therefore shall remove.